automation/hotswap.sh

```diff
@@ -14,12 +14,12 @@ fi
 FIAB=$1
 ENV=${2:-dev}
 
-printf "Generating the Leo jar. This will fail silently here if you have any compilation errors, or print a success message otherwise\n\n"
```

Comment:
does this no longer silently fail?

Reply:
yeah,  it was fixed a while ago but the message never updated